### PR TITLE
Bxc 4605 csv export parent info

### PR DESCRIPTION
**Pull request by sharonluong**
https://unclibrary.atlassian.net/browse/BXC-4605

### CODE REVIEW

File: web-services-app/src/test/java/edu/unc/lib/boxc/web/services/rest/modify/ExportCsvIT.java [hunk 1]
```diff
@@ -744,13 +742,14 @@ private void assertContainerRecord(List<CSVRecord> csvList, ResourceType objType
...
-                null, null, null, null, numChildren, described, permissions, embargoDate, "");
+                null, null, null, null, numChildren, described, permissions,
+                embargoDate, "", null, null);
     }
 
     private void assertCsvRecord(List<CSVRecord> csvList, ResourceType objType, PID expectedPid, String title,
             String path, int depth, boolean deleted, String mimetype, String checksum, Long fileSize,
             String accessSurrogate, Integer numChildren, boolean described, String permissions,
-            String embargoDate, String behavior) {
+            String embargoDate, String behavior, String parentWorkUrl, String parentWorkTitle) {
```
Comment (bbpennel) [inline web-services-app/src/test/java/edu/unc/lib/boxc/web/services/rest/modify/ExportCsvIT.java:752]: Doesn't seem like parentWorkUrl or parentWorkTitle ever get used in this method.

File: web-services-app/src/test/java/edu/unc/lib/boxc/web/services/rest/modify/ExportCsvIT.java [hunk 2]
```diff
@@ -788,6 +787,12 @@ private void assertCsvRecord(List<CSVRecord> csvList, ResourceType objType, PID
             assertEquals(permissions, rec.get(ExportCsvService.PATRON_PERMISSIONS_HEADER));
             assertEquals(embargoDate, rec.get(ExportCsvService.EMBARGO_HEADER));
             assertEquals(behavior, rec.get(ExportCsvService.VIEW_BEHAVIOR_HEADER));
+
+            if (accessSurrogate == null) {
```
Comment (bbpennel) [inline web-services-app/src/test/java/edu/unc/lib/boxc/web/services/rest/modify/ExportCsvIT.java:791]: Was this left over from another PR? It's probably fine here, I was just kind of surprised to see it as new

**Review by bbpennel — CHANGES_REQUESTED** [review]

File: web-services-app/src/test/java/edu/unc/lib/boxc/web/services/rest/modify/ExportCsvIT.java (hunk 2 above)
Comment (sharonluong) [reply web-services-app/src/test/java/edu/unc/lib/boxc/web/services/rest/modify/ExportCsvIT.java]: thanks for catching!

File: web-services-app/src/test/java/edu/unc/lib/boxc/web/services/rest/modify/ExportCsvIT.java (hunk 1 above)
Comment (sharonluong) [reply web-services-app/src/test/java/edu/unc/lib/boxc/web/services/rest/modify/ExportCsvIT.java:764]: thanks for catching!

**Pull request merged**